src/component/feature/metric-component.jsx

```diff
@@ -38,13 +38,16 @@ export default class MetricComponent extends React.Component {
         featureToggle: PropTypes.object.isRequired,
         fetchSeenApps: PropTypes.func.isRequired,
         fetchFeatureMetrics: PropTypes.func.isRequired,
+        location: PropTypes.object,
```

Comment:
I'm perfectly fine with passing the `locale` details as props as long as it is just a few components that needs it.

**But**; is there a way to set it at root level and just have a simple date-format-function that could be required when needed?